src/AC/Deal.php

```diff
@@ -0,0 +1,128 @@
+<?php
+namespace AC;
+
+class Deal extends ActiveCampaign {
+	
+	function add($params, $post_data) {
```

Comment:
Do specify visibility of methods (use `public function` instead of implicit public `function`). Try to keep code PSR-compliant: 

* Visibility MUST be declared on all properties and methods; abstract and final MUST be declared before the visibility; static MUST be declared after the visibility.
* Method names MUST be declared in `camelCase`.

There's something about consistent variable naming. I always used `camelCase` for both variables and method names. Method names are a PSR thing. IIRC, there's something in the coding standards about being consistent WRT to variable names (`$request_url` should therefore be `$requestUrl`). Given that I've been writing golang for a couple of years now, I'd write `$requestURL` now anyway.

Just have a read through https://www.php-fig.org/